# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/kasc/flightgear/hotfix-binutils-gcc.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

Hotfix needed for new binutils (bug is in gcc)
(see http://gcc.gnu.org/bugzilla/show_bug.cgi?id=16625)

--- ./src/FDM/JSBSim/FGState.h.orig	2004-09-05 15:36:43.760672000 +0200
+++ ./src/FDM/JSBSim/FGState.h	2004-09-05 15:37:01.976902712 +0200
@@ -258,6 +258,8 @@
       vLastArray[1] = vLastArray[0];
       vLastArray[0] = vTDeriv;
       break;
+    }
+    switch (type) {
     case AM3:
       vResult = (delta_t/12.0)*(  5.0 * vTDeriv
                                 + 8.0 * vLastArray[0]
